src/interface.rkt

```diff
@@ -151,4 +152,19 @@
 
 ;; Global precision tracking
 (define *output-repr* (make-parameter (get-representation 'binary64)))
-(define *var-reprs* (make-parameter '()))
\ No newline at end of file
+(define *var-reprs* (make-parameter '()))
+
+;; Herbie preprocess structs
+(struct symmetry-group (variables) #:prefab)
+
+
+;; Tracks list of preprocess structs Herbie decides to apply
+(define *herbie-preprocess* (make-parameter empty))
+
+(define (preprocess->sexp preprocess)
+  `(sort ,@(symmetry-group-variables preprocess)))
+
+(define (sexp->preprocess sexp)
+  (match sexp
+    [(list 'sort vars ...) (symmetry-group vars)]
+    [else (error (format "unknown preprocess ~a" sexp))]))
```

Comment:
Why does this live in `interface.rkt`? I'd make a new file with this and some other symmetry-related stuff.

Reply:
Good idea